src/Elliptic/Systems/Elasticity/Equations.hpp

```diff
@@ -40,9 +40,29 @@ void primal_fluxes(
         constitutive_relation,
     const tnsr::I<DataVector, Dim>& coordinates) noexcept;
 
+/*!
+ * \brief Add the contribution \f$-\Gamma^i_{ik}T^{kj} - \Gamma^j_{ik}T^{ik}\f$
```

Comment:
same concerns about how you compute `Gamma` as for the Poisson system (mostly just a note so if something needs to be changed we remember to change it in both places)